jbmc/regression/jbmc-strings/IndexOfConstantEvaluation06-AssertionFailure/test.desc

```diff
@@ -0,0 +1,9 @@
+CORE symex-driven-lazy-loading-expected-failure
+Main.class
+--function Main.constantIndexOf
+^Generated [0-9]+ VCC\(s\), 1 remaining after simplification$
+^VERIFICATION FAILED$
```

Comment:
I think you should specify which assertion you want to fail. It's probably good to use the `--property` option too, as you don't want to analyse the assertions that you expect to be successful.
Look at the tests in https://github.com/diffblue/cbmc/tree/develop/jbmc/regression/jbmc-strings/CompareToConstantEvaluation

Reply:
Yes ... it was pointed out by @danpoe when he explained to me about regression tests. I was just being lazy :sweat_smile: property added.